Model/SepaCreditTransferPaymentMethod.php

```diff
@@ -71,19 +74,24 @@ public function getTransaction()
     }
 
     /**
-     * Adds all needed data to the post-processing transaction
+     * @inheritdoc
      *
-     * @param SepaCreditTransferTransaction $oTransaction
-     * @param Transaction                   $oParentTransaction
+     * @param string      $sAction
+     * @param Transaction $oParentTransaction
      *
-     * @since 1.1.0
+     * @return SepaCreditTransferTransaction
+     *
+     * @since 1.2.0
      */
-    public function addPostProcessingTransactionData(&$oTransaction, $oParentTransaction)
+    public function getPostProcessingTransaction($sAction, $oParentTransaction)
     {
+        $oTransaction = new SepaCreditTransferTransaction();
+
```

Comment:
Is the Mandate also needed for post processing operations? is it strictly written in the user story?

Reply:
there is no user story for sepa ct.
I followed the payment sdk examples where mandate is provided to sepa ct
However I tried it without a mandate and it worked, so I will change it